themes/default/templates/home.html

```diff
@@ -267,6 +267,12 @@ <h2 class="panel-heading">{{title}}</h2>
       x[0].style.display = "block";
     }
 
+    // disables the home search button if there is no text entered to search for
+    $('.search-form').find('button[type="submit"]').on('click', function(event){
+      if($('.search-form').find('#home-search').val() == ""){
+        event.preventDefault();
+      }
+    });
```

Comment:
@willmero hey, just wondering how this would apply to white space? i.e. if the input is " " ?

Reply:
@willmero this looks good to me too! Feel free to merge! :)